### PR TITLE
Issue #74 

### PR DESCRIPTION
**Pull request by cspath1**
- [x] Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [x] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [x] Automated tests are passing
- [x] No _decreases_ in automated test coverage
- [x] Documentation updated (readme, docs, comments, etc.)
- [x] Localization: No hard-coded error messages in code files (_minimally_ in string constants)

### CODE REVIEW

**Review by KevinBusch — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -83,4 +84,4 @@
     },
     "types": "dist/index",
     "version": "0.3.4"
-}
+}
```
Comment (brandongregoryscott) [inline package.json:87]: Mind putting this new line back & squashing this down to one commit to keep the git history a little cleaner? Saw some reverts in there

**Review by brandongregoryscott — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (cspath1) [reply package.json:87]: Yep, you got it!

Comment (cspath1) [reply package.json:87]: @brandongregoryscott do you mean squashing when merging or do you want me to rebase it down to one commit?

Comment (brandongregoryscott) [reply package.json]: Ehh, now that I think about it, I can just squash it here. I think it'll erase your commits entirely and just show up as mine, though

**Review by brandongregoryscott — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (cspath1) [reply package.json]: No worries, I just finished squashing it 🙂 . Should be good to go now!

**Pull request merged**